test/services/helpers/services/users.js

```diff
@@ -13,6 +13,7 @@ const createTestUser = (app, opt) => ({
 	discoverable = false,
 	// manual cleanup, e.g. when testing delete:
 	manualCleanup = false,
+	consentStatus = 'missing',
```

Comment:
consent status should be generated from global consent check @Metauriel

Reply:
That is correct, consentStatus is passed with every get or find on consents, taking into account the age of the user.

However I dont see a problem here, since the function creates a valid consent based on the the imput consentStatus, and I understand the need to create valid constents for a user without needing to understand the consent logic yourself.
The only problem is that if a user is old enough to not require a parentconsent (or is not a student), and "parentsAgreed" is passed into the function, the consent when checked will return as "missing". That might cause confustion for the test developer, and should be communicated somehow.

Reply:
One way to communicate this is to use two boolean parameters 'withOwnConsent' and 'withParentConsent' instead of the string parameter 'consentStatus'. Then the test developer is responsible for deriving the correct consentStatus. Would this solve the problem?

Reply:
I think I would prefer an own constructor for testConsents, where you can enter all the different parts of the consent like you would with a real one

Reply:
I would prefer to create an own method to create testConsents, where you can set all the attributes just like you would when creating a real consent